src/me/mrCookieSlime/Slimefun/api/TickerTask.java

```diff
@@ -202,20 +206,29 @@ public void info(CommandSender sender) {
 			tellraw.addText("   &7&oHover for more Info");
 			StringBuilder hover = new StringBuilder();
 			int hidden = 0;
-			
-			for (Map.Entry<String, Integer> entry: machineCount.entrySet()) {
-				long timings = machineTimings.get(entry.getKey());
-				if (timings > 0) hover.append("\n&c").append(entry.getKey()).append(" - ").append(entry.getValue()).append("x &7(").append(timings).append("ms)");
-				else hidden++;
+
+			Map<String, Long> timings = machineCount.keySet().stream()
+					.map(key -> new AbstractMap.SimpleEntry<>(key, machineTimings.getOrDefault(key, 0L)))
+					.sorted((o1, o2) -> o2.getValue().compareTo(o1.getValue()))
+					.collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue, (e1, e2) -> e1, LinkedHashMap::new));
+
+			for (Map.Entry<String, Long> entry : timings.entrySet()) {
+				int count = machineCount.getOrDefault(entry.getKey(), 0);
```

Comment:
No need for "getOrDefault" here since you literally took the contents of that map to create this map, so you can safely use .get() here